Expanding examples on #514 (#1011)
MicrosoftDocs/windows-powershell-docs: docset/windows/storage/get-physicalextent.md

# Get-PhysicalExtent

## SYNOPSIS

> This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. An "extent" is the area of space on a pooled disk that contains one fragment of the data for storage space,  also can be synonymous of "allocation" or as a "slab."

Reviewer: ```suggestion
This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. An "extent" is the area of space on a pooled disk that contains one fragment of the data for storage space,  and can also be synonymous with "allocation" or "slab."
```

> This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. An "extent" is the area of space on a pooled disk that contains one fragment of the data for storage space,  and can also be synonymous with "allocation" or "slab".

Reviewer 2: ```suggestion
This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. The "extent" (also known as "allocation" or "slab)  is the area on a pooled disk containing one fragment of data for storage space,
```

## EXAMPLES

### Example 1: Get all physical extents on all physical disks
```Powershell
PS C:\>Get-PhysicalExtent -PhysicalDisk $(Get-PhysicalDisk)
```

This command gets all physical extents on all physical disks on the computer.

### Example 2: Get extents on a specific disk 
```Powershell
PS C:\>Get-PhysicalExtent -PhysicalDisk (Get-PhysicalDisk -FriendlyName "PhysicalDisk4")
```

This command gets all physical extents on the physical disk named PhysicalDisk4.
The command uses **Get-PhysicalDisk** to obtain PhysicalDisk4.

### Example 3: Get all physical extents on a disk (Other Version)
```Powershell
PS C:\>Get-PhysicalDisk -FriendlyName "PhysicalDisk4" | Get-PhysicalExtent
```

> This command gets all physical extents on the physical disk named PhysicalDisk4, using powershell pipe.

Reviewer 2: ```suggestion
This command gets all physical extents on the physical disk named PhysicalDisk4, using PowerShell pipe.
```
